adapters/nextmillennium/nextmillennium.go

```diff
@@ -126,8 +120,12 @@ func createBidRequest(prebidBidRequest *openrtb2.BidRequest, params *openrtb_ext
 
 		placementID = fmt.Sprintf("g%s;%s;%s", params.GroupID, size, domain)
 	}
-
-	bidRequest.Ext.Prebid.StoredRequest.ID = placementID
+	ext := nextMillJsonExt{}
+	ext.Prebid.Storedrequest.ID = placementID
+	jsonExt, _ := json.Marshal(ext)
```

Comment:
Please do not ignore the error from the Marshal call. This should bubble up in case it occurs. We recognize it is highly unlikely for an error to be raised here, but we've seen it before when there are memory corruption issues and the error information helps to narrow down the problem.

Reply:
I see you check for an error now, but it's not bubbled up. I guess that's fine, but is an OpenRTB request to your bidding server considered valid without the stored request id?

Reply:
It's not **completely** valid, but good enough, as we can reverse it using domain/size/etc